cmd/operator/deploy/crds/monitoring.googleapis.com_operatorconfigs.yaml

```diff
@@ -103,6 +103,34 @@ spec:
               object represents. Servers may infer this from the endpoint the client
               submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
             type: string
+          managedAlertmanager:
+            default:
+              config:
+                key: config.yaml
+                name: managed-alertmanager-config
+            description: ManagedAlertmanager holds information for configuring the
+              managed instance of Alertmanager.
+            properties:
+              config:
+                description: Config refers to the Secret in the gmp-public namespace
+                  that holds the managed Alertmanagers config file. This secret is
+                  copied to gmp-system by the operator.
+                properties:
+                  key:
+                    description: The key of the secret to select from.  Must be a
+                      valid secret key.
+                    type: string
+                  name:
+                    description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                      TODO: Add other useful fields. apiVersion, kind, uid?'
+                    type: string
+                  optional:
```

Comment:
nit: IIRC yaml fiels are optional by default, so not sure if this is needed.

Reply:
this is generated by kubebuilder